indico/modules/events/editing/controllers/backend/editable_list.py

```diff
@@ -45,7 +45,8 @@ def _process_args(self):
         RHEditableTypeEditorBase._process_args(self)
         self.contributions = (Contribution.query
                               .with_parent(self.event)
-                              .options(joinedload('editables').selectinload('revisions').selectinload('tags'))
+                              .options(joinedload('editables').selectinload('revisions').selectinload('tags'),
+                                       joinedload('person_links'))
```

Comment:
`selectinload` here as well?

Reply:
here the `joinedload` performed better both in querie # and time..